### PR TITLE
Add Romanian font and language option

### PR DESCRIPTION
**Pull request by zenseii**
This adds ISO8859-16 font used by Romanian and adds the language as a choice in the game.

Button fonts will be added in a later PR when we know what characters are needed.
image

image

### CODE REVIEW

File: src/fheroes2/gui/ui_font.cpp [hunk 1]
```diff
@@ -3267,6 +3455,9 @@ namespace fheroes2
         case SupportedLanguage::Turkish:
             // generateGoodCP1254ButtonFont( icnVsSprite[ICN::BUTTON_GOOD_FONT_RELEASED] );
             break;
+        case SupportedLanguage::Romanian:
+            // generateGoodISO8859-16ButtonFont( icnVsSprite[ICN::BUTTON_GOOD_FONT_RELEASED] );
```
Comment (zenseii) [inline src/fheroes2/gui/ui_font.cpp:3459]:
```suggestion
            // generateGoodISO8859_16ButtonFont( icnVsSprite[ICN::BUTTON_GOOD_FONT_RELEASED] );
```

File: src/fheroes2/gui/ui_font.cpp [hunk 2]
```diff
@@ -2427,6 +2427,190 @@ namespace
         }
     }
 
+    void generateISO8859_16Alphabet( std::vector<std::vector<fheroes2::Sprite>> & icnVsSprite )
```
Comment (zenseii) [inline src/fheroes2/gui/ui_font.cpp:2430]: I'm not sure this is the correct naming according to our standards, but the font name is ISO8859-16 and hyphen cannot be used. Having the underscore would make it easier should we decide to add more ISO8859 fonts other than nr.16

**Comment by zenseii** [comment]
Hi, @ihhub.

This PR is ready for you to review when you have the possibility. I left a comment regarding doubts I had about the function name. Thanks!

File: src/fheroes2/gui/ui_font.cpp (hunk 2 above)
Comment (ihhub) [reply src/fheroes2/gui/ui_font.cpp:2430]: It's fine to use such name.

**Comment by ihhub** [comment]
@zenseii , thanks a bunch for adding this change!

**Pull request merged**